dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
Alternatively, I could just enforce this based on the `--min` flag, but since we don't use that in production yet I'm not sure it's helpful.